cmd/flowrunner/testdata/flows/router_tests_test.json

```diff
@@ -15,18 +15,28 @@
                     "type": "run_result_changed",
                     "value": ""
                 },
+                {
+                    "category": "Other",
+                    "created_on": "2000-01-01T00:00:00.000000000-00:00",
+                    "input": "Ben Haggerty",
+                    "name": "Group Check",
+                    "node_uuid": "08d71f03-dc18-450a-a82b-496f64862a56",
+                    "step_uuid": "8720f157-ca1c-432f-9c0b-2014ddc77094",
+                    "type": "run_result_changed",
+                    "value": "Ben Haggerty"
```

Comment:
Getting failing flowserver trials because when a router goes to _Other_, we use the operand as the value. The operand for a legacy `group` membership test is `@step.value`, but for a new `has_group` test is `@contact`.. so we get different values

Reply:
Have made PR to change this in old engine https://github.com/nyaruka/rapidpro/pull/1993

Reply:
So now this PR is just some extra tests